lms/urls.py

```diff
@@ -1008,3 +1008,8 @@
 urlpatterns += [
     url(r'^api/course_home/', include('lms.djangoapps.course_home_api.urls')),
 ]
+
+# Applications app API urls
+urlpatterns += [
+    url(r'^api/applications/', include('openedx.adg.lms.applications.api_urls', namespace='applications_api')),
+]
```

Comment:
Let's please add this url in `openedx/adg/lms/urls` and it would automatically be added here.